src/mainMenu/mainMenu.js

```diff
@@ -0,0 +1,94 @@
+class MainMenu {
+
+    constructor() {
+        this.createMainMenu();
+        this.addListeners();
+    }
+
+    creatingButtonContainer (name) {
+        const mainMenuContainer = document.createElement('div');
+        mainMenuContainer.id = `${name}`;
+        mainMenuContainer.style=
```

Comment:
better way to add styles is to add mainMenuContainer.classList.add(className) and declare .className in css file